streaming/kafka-strimzi/manifests/03-auth/my-user.yaml

```diff
@@ -34,34 +41,3 @@ spec:
           - Write
           - Read
         host: "*"
----
-apiVersion: v1
-kind: Pod
```

Comment:
Is that Pod not necessary anymore?

Reply:
@aburhan gentle ping

Reply:
yes this is duplicated in another manifest